microsoft/testsuites/mdatp/check-mdatp.sh

```diff
@@ -113,5 +113,20 @@ if [ -f "$MDATP_OPT_DIR/mdatp_onboard.json" ]; then
     EXIT_CODE=$EXIT_ONBOARD_INFO_FOUND
 fi
 
+# special log line if mdatp installed and reports it is onboarded
+MDATP_ORG_ID=$(\
+    sudo command -v mdatp \
```

Comment:
the script is executed under sudo, no need sudo here.
I got error when run this command
```
MDATP_ORG_ID=$(\
    sudo command -v mdatp \
    && sudo mdatp health \
    | grep --fixed-strings 'org_id:' \
    | cut -f 2 -d ':' \
    | tr -d '[:blank:][:punct:]' \
)
sudo: command: command not found
```

Reply:
What distro was this on?

Reply:
I had used sudo because we also recommend this script for use by end users who hit this problem. If they want to see what went wrong and reproduce the failure, they can run it locally. My concern was people using the script in an environment where mdatp is not in the user path, but is in the sudo path. I guess if we want to be thorough we should check both.

It doesn't explain the command -v thing though. Let me know the distro and I'll investigate.

Reply:
ubuntu 2204